src/Controllers/Review.php

```diff
@@ -36,7 +36,14 @@ public function __construct( \Slim\Slim $slim = null ) {
 	protected function handleGet() {
 		$id = $this->request->get( 'id' );
 		$val = $this->request->get( 'val' );
-		$ret = $this->dao->insertCopyvioAssessment( $id, $val );
+
+		// TODO: disallow anonymous reviewers (likely by hiding the review buttons altogether)
```

Comment:
You can get rid of this TODO. I'm taking care of this in the route handler itself.